drivers/oled/oled_driver.h

```diff
@@ -191,6 +332,12 @@ typedef enum {
 // Returns true if the OLED was initialized successfully
 bool oled_init(oled_rotation_t rotation);
 
+// Send commands and data to screen
+bool oled_cmd(const uint8_t *data, uint16_t size);
+bool oled_cmd_P(const uint8_t *data, uint16_t size);
+bool oled_data(const uint8_t *data, uint16_t size);
```

Comment:
Hmm, after some more thinking `oled_data()` might not be the best name for this function (e.g., it conflicts with the `oled_data` variable in `keyboards/nullbitsco/tidbit/keymaps/snailmap_lite/keymap.c`.  Should we just accept the breakage and use the shorter name, or rename these functions to `oled_send_cmd()`, `oled_send_cmd_P()` and `oled_send_data()`, which have somewhat less chance to conflict?

Reply:
I'm not attached to the names, so `oled_send_*` seems fine.